community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/ast/convert/commands/ExpressionConverters.scala

```diff
@@ -90,7 +87,7 @@ object ExpressionConverters {
       case e: ast.GetDegree => e.asCommandGetDegree
       case e: ast.PrefixSeekRangeWrapper => e.asCommandStringSeekRange
       case e: ast.InequalitySeekRangeWrapper => e.asCommandValueSeekRange
-      case e: ast.AndedPropertyInequalities => e.asCommandAnds
+      case e: ast.AndedPropertyInequalities => e.asCommandAndedPropertyComparablePredicates
```

Comment:
this name is a real stumbling block. How are the `Predicates` that are `Comparable` to `Property` `anded` with `Command`?